index.html

```diff
@@ -255,5 +255,46 @@ <h3>LET’S CONNECT</h3>
         </div>
       </div>
     </section>
+    <footer>
+      <div class="contact-form"></div>
+      <h2 class="contact-me">Contact me</h2>
+      <p class="para-text">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form action="#" id="comment">
+        <input
+          type="text"
+          id="name"
+          placeholder="Kumilachew"
+          minlength="1"
+          maxlength="30"
+          required
+        />
```

Comment:
- [ ] Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).  The `action` value should be similar to `https://formspree.io/f/{form_id}`. Kindly implement it.